feature/bgp/admin_distance/otg_tests/admin_distance_test/admin_distance_test.go

```diff
@@ -91,7 +92,7 @@ func TestAdminDistance(t *testing.T) {
 		ts.MustAdjacency(t)
 	})
 
-	setupEBGPAndAdvertise(t, ts)
+	setupEBGPAndAdvertise(t, ts.DUT, ts)
```

Comment:
`ts` object already has DUT instance(ts.DUT), you don't have to pass separately

Reply:
comment addressed